File: web-admin/src/main/java/com/isystk/sample/web/admin/controller/html/photo/PhotoListForm.java
package com.isystk.sample.web.admin.controller.html.photo;

import com.isystk.sample.common.dto.Pageable;
import com.isystk.sample.web.base.controller.html.BaseSearchForm;

public class PhotoListForm extends BaseSearchForm implements Pageable {

  private static final long serialVersionUID = 7593564324192730932L;

  String name;

  public String getName() {
    return this.name;
  }

  public void setName(String name) {
    this.name = name;
  }
}

File: web-admin/src/main/kotlin/com/isystk/sample/web/admin/controller/html/photo/PhotoListForm.kt
package com.isystk.sample.web.admin.controller.html.photo

import com.isystk.sample.common.dto.Pageable
import com.isystk.sample.web.base.controller.html.BaseSearchForm

class PhotoListForm : BaseSearchForm(), Pageable {

    var name: String? = null

    companion object {
        private const val serialVersionUID = 7593564324192730932L
    }
}